manifests/config.js

```diff
@@ -579,6 +579,106 @@ module.exports.config = {
       },
     ],
   },
+  goerli: {
+    dataSources: [
+      // {
+      //   name: "v1Factory",
+      //   template: "v1Factory-ds.yaml",
+      //   address: "",
+      //   startBlock: 1,
+      // },
+      // {
+      //   name: "v2Factory",
+      //   template: "v2Factory-ds.yaml",
+      //   address: "",
+      //   startBlock: 1,
+      // },
+      {
+        name: "v21Factory",
+        template: "v21Factory-ds.yaml",
+        address: "0x72B8Bf40C8B316753a3E470689DA625759D2b025",
+        startBlock: 7103207,
+      },
+      {
+        name: "v22Factory",
+        template: "v22Factory-ds.yaml",
+        address: "0xA8680d0E43aDe8BC32eCCF6C446eCe4CA15d4258",
+        startBlock: 7104213,
+      },
+      // {
+      //   name: "minionFactory",
+      //   template: "minionFactory-ds.yaml",
+      //   address: "",
+      //   startBlock: 1,
+      // },
+      {
+        name: "v2minionFactory",
```

Comment:
@brossetti1 I don't think we need this or the minion template.  Just the safe minion v2

Reply:
@brossetti1 going to deploy this without the escrow minion for now. we can add that later.